File: templates/backend/fastapi/app/routes/health.py
from fastapi import APIRouter

from app.db import check_database

router = APIRouter()


@router.get("")
async def read_health():
    connected = await check_database()
    database = "connected" if connected else "disconnected"

    return {
        "server": "running",
        "database": database,
        "status": "healthy" if connected else "degraded",
        "api": "working",
    }

File: templates/backend/express/src/routes/health.js
import { Router } from "express";

import { checkDatabase } from "../db.js";

const router = Router();

router.get("/", async (_req, res) => {
  const connected = await checkDatabase();
  const database = connected ? "connected" : "disconnected";
  const healthy = connected;

  res.json({
    server: "running",
    database,
    status: healthy ? "healthy" : "degraded",
    api: "working",
  });
});

export default router;
